core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -3895,6 +3899,25 @@ components:
 ################################################
 # GENERAL / SHARED MODELS - specific substates #
 ################################################
+    BootLoaderModuleFieldVmBootSubstate:
+      allOf:
+        - $ref: '#/components/schemas/Substate'
+        - type: object
+          required:
+            - value
+          properties:
+            value:
+              $ref: '#/components/schemas/VmBootSubstate'
+    VmBootSubstate:
```

Comment:
This doesn't follow the naming convention. This should be `BootLoaderModuleFieldVmBootValue`